discovery-provider/src/queries/queries.py

```diff
@@ -2063,30 +2073,46 @@ def get_remixes_of(track_id):
                     Repost.is_current == True,
                     Repost.is_delete == False
                 )
-            ).filter(
+            ).outerjoin(
```

Comment:
i personally would have used a correlated subquery in the SELECT clause but this is a fine way to do it :slightly_smiling_face:

Reply:
I thought about doing it that way, but then I have to unzip the results, etc. which is maybe less pretty?